parsons/etl/tofrom.py

```diff
@@ -98,6 +98,142 @@ def to_html(
+                overwritten.
+            schema: dict
+                defines the rows field structure of the file.
+                Check fastavro `documentation`_ and Avro schema `reference`_ for details.
+            sample: int, optional
+                defines how many rows are inspected
+                for discovering the field types and building a schema for the avro file
+                when the `schema` argument is not passed. Default is 9.
+            codec: str, optional
+                The `codec` argument (string, optional) sets the compression codec used to
+                shrink data in the file. It can be 'null', 'deflate' (default), 'bzip2' or
+                'snappy', 'zstandard', 'lz4', 'xz' (if installed)
+            compression_level: int, optional
+                sets the level of compression to use with the specified codec (if the codec supports it)
+            avro_args: kwargs
+                Additionally there are support for passing extra options in the
+                argument `**avro_args` that are fowarded directly to fastavro. Check the
+                fastavro `documentation`_ for reference.
+
+        The avro file format preserves type information, i.e., reading and writing
+        is round-trippable for tables with non-string data values. However the
+        conversion from Python value types to avro fields is not perfect. Use the
+        `schema` argument to define proper type to the conversion.
+
+        The following avro types are supported by the schema: null, boolean,
+        string, int, long, float, double, bytes, fixed, enum,
+        :ref:`array <array_schema>`, map, union, record, and recursive types
+        defined in :ref:`complex schemas <complex_schema>`.
+
+        Also :ref:`logical types <logical_schema>` are supported and translated to
+        coresponding python types: long timestamp-millis, long timestamp-micros, int date,
+        bytes decimal, fixed decimal, string uuid, int time-millis, long time-micros.
+
+        Example usage for writing files::
+
+            >>> # set up a Avro file to demonstrate with
+            >>> table2 = [['name', 'friends', 'age'],
+            ...           ['Bob', 42, 33],
+            ...           ['Jim', 13, 69],
+            ...           ['Joe', 86, 17],
+            ...           ['Ted', 23, 51]]
+            ...
+            >>> schema2 = {
+            ...     'doc': 'Some people records.',
+            ...     'name': 'People',
+            ...     'namespace': 'test',
+            ...     'type': 'record',
+            ...     'fields': [
+            ...         {'name': 'name', 'type': 'string'},
+            ...         {'name': 'friends', 'type': 'int'},
+            ...         {'name': 'age', 'type': 'int'},
+            ...     ]
+            ... }
+            ...
+            >>> # now demonstrate what writing with toavro()
+            >>> import parsons import Table
```

Comment:
`import Parsons import Table` seems like it would be a syntax error? can you put them on two lines? I know these are just docs but people might copy and paste